collector/standard.go

```diff
@@ -0,0 +1,40 @@
+package collector
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	registerCollector("standard.go", defaultDisabled, NewStandardGoCollector)
+	registerCollector("standard.process", defaultDisabled, NewStandardProcessCollector)
+}
+
+type standardGoCollector struct {
+	origin prometheus.Collector
+}
+
+// NewStandardGoCollector creates standard go collector.
+func NewStandardGoCollector() (Collector, error) {
```

Comment:
do we use it in our other exporters? If we do, let's extract it to exporter_shared

Reply:
We do use a similar, but not exactly the same code. For example, mysqld_exporter has Scraper interface: https://github.com/percona/mysqld_exporter/blob/5640a08f7a9b8d2654e8a88961ce3faf21f9db07/collector/standard.go